src/server.js

```diff
@@ -11,12 +12,16 @@ export default class Server {
     this.accounts = [];
   }
 
-  async start() {
-    await this.ganache.listen(8545, (err, chain) => {
-      if (err) {
-        console.log("Error: ", err);
-      }
-    });
+  async start(_port) {
+    portfinder.basePort = _port;
```

Comment:
Can you store the port that the server ends up listening on in state as well? So both `server.startingPort` and `server.port` are accessible. Maybe refactor these lines into another method that sets the port?